### PR TITLE
Airflow v1.10.0

### PR DESCRIPTION
**Pull request by halldc**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

Replaces #16 

- updated to v1.10
- updated requirements of Airflow and all its database packages
- updated test imports
- added `export AIRFLOW_GPL_UNIDECODE=yes` to install script (see [note on this page](https://airflow.apache.org/installation.html) for more details)
- added myself as a maintainer

### CODE REVIEW

**Comment by halldc** [comment]
@conda-forge-admin, please rerender

**Comment by halldc** [comment]
`Unsatisfiable dependencies for platform osx-64: {"sqlalchemy[version='>=1.1.15,<1.2.0']", "flask[version='>=0.12.4,<0.13']", 'airflow==1.10.0=py36_0'}`

**Comment by halldc** [comment]
Blocked by:
- https://github.com/conda-forge/flask-appbuilder-feedstock/pull/10
- https://github.com/conda-forge/flask-openid-feedstock/pull/2

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,37 +26,45 @@ outputs:
       host:
         - python
         - pip
+        - docutils >=0.14,<1.0
       run:
         - python
         - alembic >=0.8.3,<0.9
         - bleach ==2.1.2
-        - configparser >=3.5.0, <3.6.0  # [py<35]
+        - configparser >=3.5.0,<3.6.0
```
Comment (ocefpaf) [inline recipe/meta.yaml]: You may want to wait for https://github.com/conda-forge/configparser-feedstock/pull/17
I'm removing `noarch` but I did not test if that is enough to fix that issue though.
(I'll investigate more tomorrow morning.)

**Pull request merged**